src/components/Table/TableBody/TableBody.jsx

```diff
@@ -51,8 +52,7 @@ const propTypes = {
     }),
   ]),
   hasRowActions: PropTypes.bool,
-  wrapCellText: PropTypes.oneOf(['always', 'never', 'auto']).isRequired,
-  truncateCellText: PropTypes.bool.isRequired,
```

Comment:
shouldn't these just be deprecated?

Reply:
Thanks for bringing this to my attention. I was under the impression that TableBody and TableHead were internal components (like the TableBodyRow and the TableCellRenderer), but are they in fact exported. The TableBody and TableHead were not actually using these props, only passing them down to the children, that no longer will use them.

Let me discuss with David how to best handle this. They should indeed be deprecated and perhaps I can add some kind of deprecation mapper to the new prop for the unlikely case where someone is directly using these props without the Table component. That way we don't have to introduce this as a breaking change.

Reply:
If there are not exported, then I'm ok with that... and to be clear, I'm not using these components directly, anyways, so, I'm ok if there are considered internal and you remove them.